tensorflow_recommenders_addons/dynamic_embedding/core/lib/hadoop_file_system/BUILD

```diff
@@ -1,19 +0,0 @@
-load("@local_config_tf//:build_defs.bzl", "DTF_VERSION_INTEGER", "D_GLIBCXX_USE_CXX11_ABI")
```

Comment:
Please confirm to remove the `lib/hadoop_file_system` directary at the same time.

Reply:
Done